protoc.Dockerfile

```diff
@@ -1,10 +1,11 @@
 # Docker image for protoc
 FROM node:17-alpine3.14
```

Comment:
Ah yep! I saw it, was just deferring to @fizx ; I guess specifically I'd like to avoid introducing a 2nd `docker-compose.yml`, so Kyle I think it's up to you if you'd like to either:

- Update the current `docker-compose.yml` to take a `$ARCH`-type arg, or
- Update the `FROM` to just always use `--platform=linux/amd64`

Fwiw here's a snippet of how we use a "sometimes-set-but-default-in-case" build arg in one of our `docker-compose.yml`s:

```
  db:
    build:
      context: .
      dockerfile: ./db.dockerfile
      args:
        TEST_PARALLELISM: ${TEST_PARALLELISM:-8}
```

Reply:
Specifying architectures in the FROM line has been a source of much pain for me when porting other code to M1/ARM.  I hope the updated solution is simple enough.

Reply:
@fizx ah sure! I just know that it's a thing (pinning an architecture in `FROM`); I haven't really used it personally / don't own an M1, so happy to defer since that's been a source of issues.

The ARCH build arg looks great, thanks!